src/render_helpers/mod.rs

```diff
@@ -233,18 +233,18 @@ pub fn render_to_vec(
     Ok(copy.to_vec())
 }
 
-#[cfg(feature = "xdp-gnome-screencast")]
 pub fn render_to_dmabuf(
     renderer: &mut GlesRenderer,
-    dmabuf: smithay::backend::allocator::dmabuf::Dmabuf,
+    dmabuf: Dmabuf,
     size: Size<i32, Physical>,
     scale: Scale<f64>,
     transform: Transform,
     elements: impl Iterator<Item = impl RenderElement<GlesRenderer>>,
-) -> anyhow::Result<SyncPoint> {
+) -> anyhow::Result<()> {
```

Comment:
Hm, why'd you remove this? Sure it's not used currently, but it may become needed, especially with explicit sync.